client/package.json

```diff
@@ -10,7 +10,6 @@
     "lint:nofix": "vue-cli-service lint --no-fix"
   },
   "dependencies": {
-    "@bizpal/open-services-bst": "^1.0.3",
```

Comment:
I think we should check in the package-lock file so this UI is always built with the same versions of packages (ie, server vs local builds), but its diff doesn't look right. Ideas?

Reply:
we have been checking in package-lock.json so far

Reply:
you would have had it when you ran `npm i`.  are there a bunch of changes beyond the dependencies of the bizpal tool? it was a huge tool with many dependencies...

Reply:
Yeah, it looks like the structure is messed up. I may be using a different version of npm... Let's look at it tomorrow.